src/App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
- [ ] Please remove unused files, you are using the components/App.js file, not this one in index.js.
Please remove it so you don't have two App.js files in your repo.